dts/arm/st/u5/stm32u595.dtsi

```diff
@@ -114,8 +114,8 @@
 	};
 
 	otghs_phy: otghs_phy {
-		/* Clock source defined by USBPHYC_SEL in */
-		compatible = "usb-nop-xceiv";
+		/* Clock source defined by ICKLK_SEL() in usbotg_hs `clocks` */
```

Comment:
I think `USBPHYC_SEL` was correct. Per RM0456, `The OTG_HS kernel clock is generated by the OTG_HS PHY. [...] Those frequencies [are] selected by the OTGHSSEL[1:0] multiplexer.`

Looking at the definitions:
https://github.com/zephyrproject-rtos/zephyr/blob/1689eaf6a4551165905979772d981ed6713acef5/include/zephyr/dt-bindings/clock/stm32u5_clock.h#L109
corresponds to `ICLKSEL`, used to `select the clock source for the OTG_FS, the USB and the SDMCC`
![RM0456 CCIPR1](https://github.com/user-attachments/assets/91f35b97-76c0-460e-8285-8f42c0ba0f50)

whereas 
https://github.com/zephyrproject-rtos/zephyr/blob/1689eaf6a4551165905979772d981ed6713acef5/include/zephyr/dt-bindings/clock/stm32u5_clock.h#L125
corresponds to `OTGHSSEL` used to `select the OTG_HS PHY kernel clock source`
![RM0456 CCIPR2](https://github.com/user-attachments/assets/f695d956-b3b9-459c-b07c-e4b592837467)

Reply:
got it, will flip it back in a sec.

Reply:
Oh interesting, does this require further device tree changes for it to be "correct", besides just the comment?

Reply:
Ah, hm, should we create a separate / duplicate entry for OTGHSSEL[1..0] in that dt file? Or if it's not being used (a quick grep shows it's not) then just rename it to OTGHSSEL ?

Reply:
Rename from `USBPHYC_SEL` to `OTGHS_SEL` sounds good to me. Doesn't look like it would conflict with anything else.

Reply:
ok, done!